pkg/config/sync.go

```diff
@@ -426,6 +432,22 @@ func mergeContainerSecurityContext(base, patch *corev1.SecurityContext) *corev1.
 	return patched
 }
 
+func mergeTLSCertificateConfigmapRef(operatorConfig, clusterConfig *controller.ConfigmapReference) *controller.ConfigmapReference {
+	mergedConfigmapReference := &controller.ConfigmapReference{
```

Comment:
Now that we've removed `controller.ConfigmapReference{}` from the DWOC, you'll have to use `types.NamespacedName` here as well, and change this function's parameters.

Reply:
done